src/studio/src/designer/backend/Controllers/RepositoryController.cs

```diff
@@ -563,12 +572,27 @@ public ActionResult ContentsZip(string org, string repository, [FromQuery] bool
                     archive.CreateEntryFromFile(Path.Join(appRoot, changedFile), changedFile);
                 }
             }
-            
+
             outStream.Seek(0, SeekOrigin.Begin);
 
             return File(outStream, "application/zip", $"{org}-{repository}.zip");
         }
 
+        /// <summary>
+        /// Gets the repository type
+        /// </summary>
+        /// <param name="org">Organization owning the repository identified by it's short name.</param>
+        /// <param name="repository">Repository name to check.</param>
+        [HttpGet]
+        [Route("/designer/api/v1/{org}/{repository}/repos/repositoryType")]
```

Comment:
"designer/api/{org}/{repository}/datamodels" is defined as a route on controller level.
I guess here relative url should be used.